docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+# Kubeflow Release Handbook
+
+## Executive Summary
+
+The purpose of this document is to define the roles and processes for the Kubeflow community to release Kubeflow. This document will attempt to coordinate releasing between the five major components and working groups in Kubeflow, as well as defining best practices, roles, and processes to make releasing Kubeflow easier for the community with each iteration.
+
+## People and Roles
+
+### Release Manager
+The Release Manager will be responsible for coordinating the release and taking ultimate accountability for all release tasks to be completed on time.
+
+**Prerequisites:**
+1. Be a contributor for at least 2 years
+2. Have shadowed in a previous release or be a Release Team member in the past
+3. Contributions in at least 2 Working Groups
+
+**Time Commitments:** This role will need to be synchronous. The Release Manager is expected to be joining all the weekly and burndown meetings as well as coordinate with the Release Team and the Working Group leads. For the Development phase an estimate of the work needed is 4-8 hours per week. For the next phases the Release Manager will need to devote more time and coordinate the tasks of the release.
```

Comment:
Part of the idea behind having a release team was to distribute the load among multiple people, ideally in different time zones, so that one of the release managers would be present in every meeting. This way, there isn't a single person that needs to attend all meetings, as some will be in the middle of the night most likely, and that the working groups could coordinate things critical for the release with a release manager regardless of the time zone to make things move more smoothly. 
While the release managers for this release are (sadly) all in the same time zone area, they can coordinate among them how to distribute night shifts.

Reply:
+1

Reply:
This is why the release team will have members, so that if the release manager cannot make it in a meeting in a timezone then other members can give the updates to these meetings.